protocol/0082-ETHD-ethereum-data-source.md

```diff
@@ -111,80 +111,81 @@ Select {
 
 1. Using the existing ways to create or update a market via governance proposals, define data sources for settlement and termination as the result of calling a read method of a smart contract on ethereum network. (Phase 2 of this step would be defining an oracle that is based on listening for events on ethereum network) (<a name="0082-ETHD-001" href="#0082-ETHD-001">0082-ETHD-001</a>)
-2. All current governance rules that apply to propose / submit / vote on a proposal should be applicable for the ethereum oracle data source creation / amendment  (<a name="0082-ETHD-002" href="#0082-ETHD-002">0082-ETHD-002</a>)
-3. Create more than spam.protection.max.proposals oracle data source proposals in an epoch - proposal rejected with error message (<a name="0082-ETHD-003" href="#0082-ETHD-003">0082-ETHD-003</a>)
-4. Create ethereum oracles based on calling a read method of a smart contract by supplying incorrect ABI (Phase 2 - oracle based on listening for events) (<a name="0082-ETHD-004" href="#0082-ETHD-004">0082-ETHD-004</a>)
 
 ### External Oracles - Amendments
 
-1. Amend the oracle data source via governance proposals. Amendments should take effect as soon as the proposal is enacted (<a name="0082-ETHD-005" href="#0082-ETHD-005">0082-ETHD-005</a>)
-2. Amend an existing ethereum data source and change the contract address and enact the proposal. Once enacted, the data should be sourced from the new smart contract. Try amending the other fields in the oracle data source and the changes should take effect after the proposals are enacted (<a name="0082-ETHD-006" href="#0082-ETHD-006">0082-ETHD-006</a>)
-3. Phase 2 - Amend an existing ethereum data source and change the events that we are listening to and enact the proposal. Once enacted , the data should be sourced from the amended events. Try amending the other fields in the oracle data source and the changes should take effect after the proposals are enacted (<a name="0082-ETHD-007" href="#0082-ETHD-007">0082-ETHD-007</a>)
+1. Update an existing market using the market update proposal to change the smart contract address and read method. The changes take effect after the market update proposal is enacted and data is sourced from the new smart contract. The old data source  will be deactivated whhen the proposal is enacted (<a name="0082-ETHD-005" href="#0082-ETHD-005">0082-ETHD-005</a>)
+2. Using the market update proposal all data elements for the ethereum oracles can be updated. On successful enactment , a new oracle data source is created for the market. In case any existing data source matches the new data source , then a new data source is not created and the existing one is used.
+3. Phase 2 - Update an existing market using the market update proposal to change the events that the market is listening to. The changes take effect after the market update proposal is enacted and data is sourced from the new events. (<a name="0082-ETHD-007" href="#0082-ETHD-007">0082-ETHD-007</a>)
```

Comment:
If this is not for October but a later phase we can mark the ACs like this so we can track whats really needed for October but not lose the ACs

```suggestion
3. Phase 2 - Update an existing market using the market update proposal to change the events that the market is listening to. The changes take effect after the market update proposal is enacted and data is sourced from the new events. (<a name="0082-PLAZZO-007" href="#0082-PLAZZO-007">0082-PLAZZO-007</a>)
```

Reply:
AC numbering updated.